commerce/domain/checkout/order/createOrder/classes/CreateOrderSampleIntegrationTest.cls

```diff
@@ -0,0 +1,68 @@
+/*
+ * @description Example of "integration" test for CreateOrder extension point. This test is calling real default implementation of CreateOrder Extension.
+ * Please be aware that this test might fail in some orgs in case when default implementation suppose to call other services that are not properly set up / stubbed for the test.
+ */
+@isTest
+public class CreateOrderSampleIntegrationTest {
+    @isTest
+    public static void shouldRoundUpCostOfProduct() {
+        CartExtension.Cart cart = arrangeCart(new Decimal[]{1.5});
+        CreateOrderSample createOrder = new CreateOrderSample();
+        // Act
+        Test.startTest();
+        CartExtension.CreateOrderResponse response = createOrder.createOrder(new CartExtension.CreateOrderRequest(cart, '123'));
+        Test.stopTest();
+        // Assert
+        Order order = response.getOrderGraph().getOrder();
+        Assert.areEqual('Total rounded up cost of products: 2.0', order.Description);
+    }
+
+    @isTest
+    public static void shouldRoundUpCostOfSeveralProducts() {
+        CartExtension.Cart cart = arrangeCart(new Decimal[]{1.5, 2.6});
+        CreateOrderSample createOrder = new CreateOrderSample();
+        // Act
+        Test.startTest();
+        CartExtension.CreateOrderResponse response = createOrder.createOrder(new CartExtension.CreateOrderRequest(cart, '123'));
+        Test.stopTest();
+        // Assert
+        Order order = response.getOrderGraph().getOrder();
+        Assert.areEqual('Total rounded up cost of products: ' + (2.0 + 3.0), order.Description);
+    }
+
+        private static CartExtension.Cart arrangeCart(Decimal[] itemsSalePrice) {
```

Comment:
Indentation does not look right for this function

Reply:
fixed